shared/src/business/entities/EntityConstants.js

```diff
@@ -435,6 +436,18 @@ const TRACKED_DOCUMENT_TYPES = {
   },
 };
 
+const SINGLE_DOCKET_RECORD_ONLY_EVENT_CODES = flatten([
+  ...Object.values(DOCUMENT_INTERNAL_CATEGORIES_MAP),
+])
+  .filter(internalEvent => internalEvent.caseDecision)
+  .map(x => x.eventCode);
+
+const MULTI_DOCKET_FILING_EVENT_CODES = flatten([
+  ...Object.values(DOCUMENT_INTERNAL_CATEGORIES_MAP),
+])
+  .filter(internalEvent => !internalEvent.caseDecision)
+  .map(x => x.eventCode);
```

Comment:
:camping: I wonder if we should consider a TypeScript class or interface for event codes at this point. It might be easier to manage if there are functions on a type to get back a specific subset that we need.